modules/console/pages/flow-console.adoc

```diff
@@ -32,6 +32,10 @@ When enabled, the Skupper console URL is displayed whenever you check site statu
 +
 --
 Enabling the console also requires that you enable the flow-collector component, which requires resources to process traffic data from all sites.
+
+NOTE: To display components that require services rather than provide services, the flow-collector makes all services that are available in the namespace visible in the console.
```

Comment:
Maybe we can remove this statement "To display components that require services rather than provide services". IMO. This sentence seems a criteria to follow instead a note